File: competition-serv-model/src/main/java/compman/compsrv/model/commands/payload/MoveCompetitorPayload.java
package compman.compsrv.model.commands.payload;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
public class MoveCompetitorPayload implements Serializable {

    private String competitorId;
    private String sourceFightId;
    private String targetFightId;
    private int index;
}

File: competition-serv-service/src/main/java/compman/compsrv/model/es/commands/payload/MoveCompetitorPayload.kt
package compman.compsrv.model.es.commands.payload

import compman.compsrv.model.competition.Competitor
import compman.compsrv.model.dto.CategoryDTO

data class MoveCompetitorPayload(val competitorId: String, val sourceFightId: String, val targetFightId: String, val index: Int)